Failing ci-benchmark-scheduler-perf-master tests for PreemptionAsync and Unschedulable tests

### Which jobs are failing?

ci-benchmark-scheduler-perf-master

### Which tests are failing?

PreemptionAsync and Unschedulable test cases

### Since when has it been failing?

17th Oct 2024

### Testgrid link

https://testgrid.k8s.io/sig-scalability-benchmarks#scheduler-perf

### Reason for failure (if possible)

PreemptionAsync test is failing because of context deadline error:
```
    scheduler_perf.go:1427: FATAL ERROR: op 3: error in waiting for pods to get scheduled: at least pod namespace-3/pod-4vfnq is not scheduled: context deadline exceeded
--- FAIL: BenchmarkPerfScheduling/PreemptionAsync/5000Node
```
Caused by #127829

Unschedulable test is failing because of too high threshold configured:
```
    scheduler_perf.go:1298: ERROR: op 2: BenchmarkPerfScheduling/Unschedulable/5kNodes/10kPods/namespace-2: expected SchedulingThroughput Average to be higher: got 289.204988, want 400.000000
--- FAIL: BenchmarkPerfScheduling/Unschedulable/5kNodes/10kPods
```
Caused by #128153
After making these changes, the threshold should be even lower, as it seems a value around 270-280 should be good at the moment.  

### Anything else we need to know?

You should ignore the first row in testgrid (k8s.io/kubernetes/test/integration/scheduler_perf.scheduler_perf), as it doesn't affect the results and check only the second one (ci-benchmark-scheduler-perf-master.Overall).

### Relevant SIG(s)

/sig scheduling

--- a/test/integration/scheduler_perf/config/performance-config.yaml
+++ b/test/integration/scheduler_perf/config/performance-config.yaml
@@ -688,8 +688,8 @@
 #      initPods: 20000
 #      measurePods: 5000
 
-# Measure throughput of regular schedulable pods that are interleaved by high priority preemption pods scheduled at 20/s rate.
-# Implementation of asynchrounous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the trhoughput of the measured pods as all heavy operations (apiserver communication) will be performed asyncronously, wihtout blocking the scheduler loop.
+# Measure throughput of regular schedulable pods that are interleaved by high priority preemption pods scheduled at 5/s rate.
+# Implementation of asynchronous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the throughput of the measured pods as all heavy operations (apiserver communication) will be performed asynchronously, without blocking the scheduler loop.
 # How is it achieved:
 # 1. There are X initial nodes with 4 low priority pods each, consuming 3.6 CPU out of the total 4 available.
 # 2. High priority preemption which need to preempt 3 of 4 low priority pods to fit (require 3 CPU).
@@ -705,7 +705,7 @@
     mode: create
     templatePaths:
     - config/templates/pod-high-priority-large-cpu.yaml
-    intervalMilliseconds: 50
+    intervalMilliseconds: 200
   - opcode: createPods
     countParam: $measurePods
     podTemplatePath: config/templates/pod-default.yaml
@@ -725,7 +725,7 @@
       measurePods: 500
   - name: 5000Nodes
     labels: [performance]
-    threshold: 200
+    threshold: 120
     params:
       initNodes: 5000
       initPods: 20000
@@ -763,7 +763,7 @@
       measurePods: 1000
   - name: 5kNodes/10kPods
     labels: [performance]
-    threshold: 400
+    threshold: 250
     params:
       initNodes: 5000
       measurePods: 10000

